.travis.yml

```diff
@@ -8,4 +8,6 @@ branches:
     - master
 
 script:
-- ./test.bash
+- curl https://raw.githubusercontent.com/GoogleCloudPlatform/runtimes-common/master/structure_tests/ext_run.sh > ext_run.sh
```

Comment:
I suppose you could pin this to a specific commit hash, but in general we don't really want people using anything except master/latest. We'll be releasing new versions of the image continuously and be able to rollback should anything break. But if you really wanted to nothing is stopping you from replacing `master` with a commit hash you know works.